.gitignore

```diff
@@ -9,3 +9,5 @@ phpunit.xml
 /phpcs.xml
 /.phpcs-cache
 /box.phar
+/.idea
```

Comment:
IMO, this line belongs to a personal `.gitignore` file but not to the project, unless the project is tightly coupled to the IDE.

Reply:
I didn't know you could do that. I will remove it and do that. Thanks!

Reply:
Removed here https://github.com/doctrine/migrations/commit/d3d7a8ce0e5133c7c48b46cbaf9ee0ee48191492

Reply:
> IMO, this line belongs to a personal `.gitignore` file 

:+1: 

There is nice article from GitHub: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore